### PR TITLE
Create doubly_linked_list.py

### PR DESCRIPTION
**Pull request by keshavgbpecdelhi**
Title : Created Create doubly_linked_list.py [append some items and iterate through the list (print forward)]
Fixes : No fixes just created
Closes : #56 
Details - 

- Created a class Node which requires object as a parameter in which creating an `__init__` function for value initialization :

- `class Node(object):`
- ` def __init__(self, data=None, next=None, prev=None): `

- Then finally creating a class named `doubly_linked_list(object):` which was required to be done in this issue.

Test - test attachments  are included in this.

Screenshot (233)

Screenshot (232)

### CODE REVIEW

**Comment by devkapilbansal** [comment]
@keshavgbpecdelhi please link issue under description as we already in template.

File: dsalgo/doubly_linked_list.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class Node(object):
+    # Doubly linked node
```
Comment (devkapilbansal) [inline dsalgo/doubly_linked_list.py:2]: Add description about your changes here

Comment (keshavgbpecdelhi) [reply dsalgo/doubly_linked_list.py:2]: Where to write sir?

Comment (keshavgbpecdelhi) [reply dsalgo/doubly_linked_list.py:2]: > @keshavgbpecdelhi please link issue under description as we already in template.

#56 done

File: dsalgo/doubly_linked_list.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+class Node(object):
```
Comment (keshavgbpecdelhi) [inline dsalgo/doubly_linked_list.py:1]: create a doubly linked list, append some items and iterate through the list

File: dsalgo/doubly_linked_list.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    # Doubly linked node
```
Comment (keshavgbpecdelhi) [inline dsalgo/doubly_linked_list.py:2]: It is a comment written to show initialization of doubly linked node i.e.
    def __init__(self, data=None, next=None, prev=None):
        self.data = data
        self.next = next
        self.prev = prev

**Comment by devkapilbansal** [comment]
@keshavgbpecdelhi please use pull request template specified

**Comment by devkapilbansal** [comment]
Pinging @17sushmita for reviewing code

**Comment by keshavgbpecdelhi** [comment]
> @keshavgbpecdelhi please use pull request template specified

@devkapilbansal Sir now edited.

**Pull request closed without merging**